packages/plugin-core/src/commands/CopyAsCommand.ts

```diff
@@ -0,0 +1,78 @@
+import { assertUnreachable } from "@dendronhq/common-all";
+import {
+  CopyAsFormat,
+  getAllCopyAsFormat,
+  JSONV2PodConfig,
+  MarkdownV2PodConfig,
+  PodExportScope,
+  PodV2Types,
+} from "@dendronhq/pods-core";
+import _ from "lodash";
+import { PodCommandFactory } from "../components/pods/PodCommandFactory";
+import { PodUIControls } from "../components/pods/PodControls";
+import { DENDRON_COMMANDS } from "../constants";
+import { VSCodeUtils } from "../vsCodeUtils";
+import { BasicCommand, CodeCommandInstance } from "./base";
+
+type CommandOutput = void;
+type CommandOpts = CodeCommandInstance;
+
+/**
+ * Command that will find the appropriate export command to run, and then run
+ * it. This is the UI entry point for all export pod functionality.
+ */
+export class CopyAsCommand extends BasicCommand<
+  CommandOpts,
+  CommandOutput,
+  CopyAsFormat
```

Comment:
Can we keep `CommandInput` like we usually do and give it a property `format`?

as-is, keybindings have to be defined like this:

```
  {
    "command": "dendron.copyAs",
    "key": "cmd+k m",
    "when": "dendron:pluginActive",
    "args": "Markdown"
  }
```

Instead I think we want this:

```
  {
    "command": "dendron.copyAs",
    "key": "cmd+k m",
    "when": "dendron:pluginActive",
    "args": {
        "format": "Markdown"
    }
  }
```